components/sensor/kalman_combinator.rst

```diff
@@ -0,0 +1,61 @@
+Kalman filter-based sensor fusion
+=================================
+
+.. seo::
+    :description: Instructions for setting up a kalman_combinator sensor
+
+The ``kalman_combinator`` sensor platform allows you to filter one or several
+sensors into one with a reduced error. If using a single sensor as data source,
+it acts like a :ref:`sensor-filter-exponential_moving_average` filter. With
+multiple sensors, it combines their values based on their respective standard
+deviation.
+
+The `unit_of_measurement`, `device_class`, `entity_category`, `icon`, and
+`accuracy_decimals` properties are by default inherited from the first sensor.
+`state_class` is explicitly not inherited, because `total_increasing` states
```

Comment:
```suggestion
The ``unit_of_measurement``, ``device_class``, ``entity_category``, ``icon``, and
``accuracy_decimals`` properties are by default inherited from the first sensor.
``state_class`` is explicitly not inherited, because ``total_increasing`` states
```